### PR TITLE
Generalize message in specextract

### PR DESCRIPTION
**Pull request by hamogu**
This addresses the first point in #843.

### CODE REVIEW

File: ciao_contrib/_tools/specextract.py [hunk 1]
```diff
@@ -443,7 +443,7 @@ def _check_event_stats(self,file,refcoord_check=None,
         if counts == "0":
             try:
                 if refcoord_check.lower() in ["","none","indef"]:
-                    raise IOError(f"{file} has zero counts. Check that the region format is in sky pixels coordinates.")
+                    raise IOError(f"{file} has zero counts. Check that the region correct (e.g. wrong region, coordinates not in sky pixels or degrees).")
```
Comment (hamogu) [inline ciao_contrib/_tools/specextract.py:446]: Do we want to put the workaround to run this anyway in the message or do we thing it's better to leave that for expert users to figure out themselves on the thought that in most cases zero counts actually indicates an error in the region?

```suggestion
                    raise IOError(f"{file} has zero counts. Check that the region correct (e.g. wrong region, coordinates not in sky pixels or degrees) or use parameter "refcoord"")
```

Comment (DougBurke) [reply ciao_contrib/_tools/specextract.py:446]: Let's be explicit...

Comment (DougBurke) [inline ciao_contrib/_tools/specextract.py:446]:
```suggestion
                    raise IOError(f"{file} has zero counts. Check that the region correct (e.g. wrong region, coordinates not in sky pixels or degrees) or use parameter='refcoord'.")
```

Comment (DougBurke) [reply ciao_contrib/_tools/specextract.py]: Redid the suggestion to fix a typo (use of `"` rather than `'`)

**Pull request merged**